### ISSUE TITLE
.

### ISSUE THREAD

**Issue by ghost** [domoticz/domoticz#3805]
.

**Comment by rwaaren** [comment]
> I've setup /etc/init.d/domoticz.sh with the following:
> DAEMON_ARGS="$DAEMON_ARGS -wwwbind 192.168.5.20"
> I expect it to access 192.168.5.20:80 instead of 127.0.0.1:80 as the whole rest of domoticz seems to do it. 

Check [Using dzVents with domoticz](https://www.domoticz.com/wiki/DzVents:_next_generation_LUA_scripting#Using_dzVents_with_Domoticz) where you will read that dzVents needs access to 127.0.0.1 and why.

The github issues are not the right place for feature requests. Please use the forum

**Comment by Catfriend1** [comment]
Thanks for the information. I'm fine with the 127.0.0.1 "system requirement". Did a workaround to solve it by -wwwbind'ing to 127.0.0.1 and then using iptables to forward port 80 request from the 192.168.5.20 network interface only. Will use forum next time.

/etc/init.d/domoticz.sh
```
DAEMON_ARGS="$DAEMON_ARGS -wwwbind 127.0.0.1"
iptables -t nat -I PREROUTING -p tcp -i eth0 -d 192.168.5.20 --dport 80 -j DNAT --to-destination 127.0.0.1:80
sysctl -w net.ipv4.conf.eth0.route_localnet=1
```

**Comment by Catfriend1** [comment]
Ref.: https://www.domoticz.com/forum/viewtopic.php?f=31&t=30610

**Comment by rwaaren** [comment]
Should be fixed in #4598

**Comment by Catfriend1** [comment]
Thanks! :-)
